### ISSUE TITLE
[App Support] After 2.1.0 update unable to give it keyboard access 

### ISSUE THREAD

**Issue by rajeshneo** [beardedspice/beardedspice#481]
Could be my system issue but when invoking accessibility i get following error
"Security agent may only be invoked by apple software" 

Did we added new permissions?

**Comment by Stillness-2** [comment]
No. May be this relates with changing of the developer certificate. I don't know. In my case, request of accessibility went in normal mode.

**Comment by rajeshneo** [comment]
Fixed after restart.. most likely local policy issue..
